### PR TITLE
Add time accounting data for program and obs

### PR DESCRIPTION
**Pull request by hugo-vrijswijk**
Program tab doesn't _really_ show changes for me as it had 0 as a placeholder before and shows 0 from the API.

Note the durations in the tile title:

afbeelding

afbeelding

### CODE REVIEW

**Review by toddburnside — APPROVED** [review]
I don't know much about the time accounting stuff, but LGTM.

File: explore/src/main/scala/explore/config/sequence/SequenceEditorTile.scala [hunk 1]
```diff
@@ -3,31 +3,17 @@
...
 object SequenceEditorTile:
 
-  def sequenceTile(
-    programId:       Program.Id,
-    obsId:           Observation.Id,
-    targetIds:       List[Target.Id],
-    snPerClass:      Map[ObserveClass, SignalToNoise],
-    sequenceChanged: View[Pot[Unit]]
-  ) =
+  def sequenceTile =
```
Comment (rpiaggio) [inline explore/src/main/scala/explore/config/sequence/SequenceEditorTile.scala:14]: Can this be a `val` now?

File: explore/src/main/scala/explore/tabs/ObsTabTiles.scala [hunk 2]
```diff
@@ -361,12 +363,28 @@ object ObsTabTiles:
...
-              sequenceChanged
+            SequenceEditorTile.sequenceTile(renderInTitle =>
+              React.Fragment(
+                renderInTitle {
+                  val programTimeCharge = props.observation.get.execution.timeCharge.program
+                  props.observation.get.executionTime
+                    .map { planned =>
+                      val total = programTimeCharge +| planned
+                      <.span(
+                        <.span(^.fontWeight := "bold", total.toHoursMinutes),
```
Comment (rpiaggio) [inline explore/src/main/scala/explore/tabs/ObsTabTiles.scala:374]: `^.fontWeitght.bold`. Can we move styling to a class?

File: model/shared/src/main/scala/explore/model/syntax/package.scala [hunk 3]
```diff
@@ -75,7 +75,12 @@ object all:
      * Format a timespan in the format `${hh}hrs ${mm}mins`
      */
     def toHoursMinutes: String =
-      s"${timespan.toHours.toLong}hrs ${(timespan.toMinutes.toLong) % 60}mins"
+      val hours   = timespan.toHours.toLong
+      val minutes = (timespan.toMinutes.toLong) % 60
+
+      if hours === 0 then s"${minutes}mins"
+      else if minutes === 0 then s"${hours}hrs"
+      else s"${hours}hrs ${minutes}mins"
```
Comment (rpiaggio) [inline model/shared/src/main/scala/explore/model/syntax/package.scala:83]: Nice

**Review by rpiaggio — APPROVED** [review]
LGTM

File: explore/src/main/scala/explore/config/sequence/SequenceEditorTile.scala (hunk 1 above)
Comment (hugo-vrijswijk) [reply explore/src/main/scala/explore/config/sequence/SequenceEditorTile.scala]: Sure. Don't think it makes much of a difference, but it is now a val

File: explore/src/main/scala/explore/tabs/ObsTabTiles.scala (hunk 2 above)
Comment (hugo-vrijswijk) [reply explore/src/main/scala/explore/tabs/ObsTabTiles.scala]: Moved it to a class

**Pull request merged**